app/kumactl/cmd/install/install_control_plane.go

```diff
@@ -52,12 +55,16 @@ func newInstallControlPlaneCmd(pctx *kumactl_cmd.RootContext) *cobra.Command {
 		SdsTlsKey:               "",
 		CNIImage:                "lobkovilya/install-cni",
 		CNIVersion:              "0.0.1",
+		KumaCpMode:              core.StandAlone,
 	}
 	cmd := &cobra.Command{
 		Use:   "control-plane",
 		Short: "Install Kuma Control Plane on Kubernetes",
 		Long:  `Install Kuma Control Plane on Kubernetes in a 'kuma-system' namespace.`,
 		RunE: func(cmd *cobra.Command, _ []string) error {
+			if err := core.ValidateCpMode(args.KumaCpMode); err != nil {
+				return err
+			}
```

Comment:
Do I have to provide the mode now or can I still do `kumactl install control-plane` and standalone is by default?

Reply:
standalone is configured as a default value